test/jdk/javax/swing/TestDefListModelExcpn.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+/*
+ * @test
+ * @bug 6187113
+ * @summary  Verifies if
+ *  DefaultListSelectionModel.removeIndexInterval(0, Integer.MAX_VALUE) fails
+ * @run main TestDefListModelExcpn
+ */
+import javax.swing.DefaultListSelectionModel;
+
+public class TestDefListModelExcpn {
+    public static void main(String[] args) throws Exception {
```

Comment:
@prsadhuk A more descriptive exception message using try catch block would be more helpful than a generic throws Exception on the main method.

Reply:
I am not sure it will, it is throwing IOOBE with the statcktrace by default and we normally relied on that for such exceptions AIOBE, IOOBE, CCE etc, not needed to catch and rethrow